app/src/main/res/layout/activity_main.xml

```diff
@@ -15,23 +15,17 @@
         android:name="com.parishod.watomatic.fragment.MainFragment"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        app:layout_constraintBottom_toTopOf="@+id/guideline2"
-        app:layout_constraintTop_toTopOf="parent"/>
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toTopOf="@+id/branding_frame_layout"
+        />
 
     <androidx.fragment.app.FragmentContainerView
         android:id="@+id/branding_frame_layout"
         android:name="com.parishod.watomatic.fragment.BrandingFragment"
         android:layout_width="match_parent"
-        android:layout_height="50dp"
```

Comment:
Hardcoded height was squishing it when text is enlarged or multi line text is used so replaced with "wrap_content"